debugtools/DDR_VM/src/com/ibm/j9ddr/view/dtfj/image/J9DDRImageProcess.java

```diff
@@ -317,13 +298,13 @@ private String getExecutablePathFromSystemProperties() {
 							if (m.find()) {
 								String javaHome = m.group(1);
 
-								// Note this method not used on Windows - don't have to worry about .exe or \
+								// Note this method not used on Windows - don't have to worry about .exe or backslashes.
```

Comment:
"method is not used" instead of "method not used"

Reply:
Fixed in faa28329a2cb1304baddf0f9ff17d81a8b45aaa1.